maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/IntegrationTestMojo.java

```diff
@@ -34,10 +34,11 @@
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.plugins.annotations.ResolutionScope;
-import org.apache.maven.shared.utils.ReaderFactory;
 import org.apache.maven.shared.utils.StringUtils;
 import org.apache.maven.surefire.suite.RunResult;
 
+import static org.apache.maven.shared.utils.ReaderFactory.FILE_ENCODING;
+
```

Comment:
is it really intentional to reformat everything to have static import on any static access? I don't find it convenient in general.